core/src/main/scripts/importer/validateData.py

```diff
@@ -1792,6 +1792,15 @@ def checkHeader(self, cols):
                     extra={'line_number': self.line_number,
                            'column_number': col_index + 1,
                            'cause': col_name})
+            # do not allow special characters in attribute names
+            if not re.match(
+                    # regex to allow only alpha numerics and _ underscore:
+                    r'^[A-Za-z0-9_]+$', col_name):
```

Comment:
should we support `[^A-Za-z0-9\._-]`?

Reply:
@jjgao any specific public studies you have in mind?